docs/source/nlp/question_answering_squad.rst

```diff
@@ -0,0 +1,301 @@
+   - if the question is not answerable, an empty ``answers`` list is provided
+
+.. _SQuAD format: https://rajpurkar.github.io/SQuAD-explorer/
+
+The evaluation files (for validation and testing) follow the above format, except that it can provide more than one answer to the 
+same question. The inference file also follows the above format, except that it does not require the ``answers`` and ``is_impossible`` 
+keywords.
+
+The following is an example of the data format (JSON file):
+
+.. code::
+
+    {
+        "data": [
+            {
+                "title": "Super_Bowl_50",
+                "paragraphs": [
+                    {
+                        "context": "Super Bowl 50 was an American football game to determine the champion of the National Football League (NFL) for the 2015 season. The American Football Conference (AFC) champion Denver Broncos defeated the National Football Conference (NFC) champion Carolina Panthers 24\u201310 to earn their third Super Bowl title. The game was played on February 7, 2016, at Levi's Stadium in the San Francisco Bay Area at Santa Clara, California. As this was the 50th Super Bowl, the league emphasized the \"golden anniversary\" with various gold-themed initiatives, as well as temporarily suspending the tradition of naming each Super Bowl game with Roman numerals (under which the game would have been known as \"Super Bowl L\"), so that the logo could prominently feature the Arabic numerals 50.",
+                        "qas": [
+                            {
+                                "question": "Where did Super Bowl 50 take place?",
+                                "is_impossible": "false",
+                                "id": "56be4db0acb8001400a502ee",
+                                "answers": [
+                                    {
+                                        "answer_start": "403",
+                                        "text": "Santa Clara, California"
+                                    }
+                                ]
+                            },
+                            {
+                                "question": "What was the winning score of the Super Bowl 50?",
+                                "is_impossible": "true",
+                                "id": "56be4db0acb8001400a502ez",
+                                "answers": [
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
+        ]
+    }
+
+
+Dataset Download
+----------------
+
+To perform training of the Question Answering model on the SQuAD dataset, you must first download it from `here
+<https://rajpurkar.github.io/SQuAD-explorer/>`_ or run:
+
+.. code::
+
+    python get_squad.py 
+
+There are two versions: 
+
+- SQuAD version 1.1, which does not contain questions without the answer and has 100,000+ question-answer pairs on 500+ articles.
```

Comment:
Keep something like this in your new documentation - also do the same for MSMarco if it's straightforward